sphinxcontrib/drawio.py

```diff
@@ -268,6 +272,7 @@ def setup(app: Sphinx) -> Dict[str, Any]:
     app.add_directive("drawio", DrawIO)
     app.add_config_value("drawio_output_format", "png", "html", ENUM(*VALID_OUTPUT_FORMATS))
     app.add_config_value("drawio_binary_path", None, "html")
+    app.add_config_value("drawio_default_scale", 1, "html", directives.positive_int)
```

Comment:
I do not think `directives.positive_int` works here in the config values, it only supports an ENUM of output formats.

Reply:
I thought I saw it taking a function/lamda? ill check again and update as needed